lib/localenv/localenv.go

```diff
@@ -319,6 +319,15 @@ func (env *LocalEnvironment) OperatorService(opsCenterURL string, options ...htt
 	return client, nil
 }
 
+// SiteOperatorWithTimeout returns Operator for the local gravity site with a
+// timeout duration.
+func (env *LocalEnvironment) SiteOperatorWithTimeout(duration time.Duration) (*opsclient.Client, error) {
```

Comment:
nit: we've been moving away from `Site` in favor of `Cluster` for some time now - would be cool to avoid `[Ss]ite` in new method names.
Another question - why not hardcode a timeout in the service clients universally - is there a reason to have a client with no timeout in the first place?

Reply:
Good point, I've modified the GetClient func to use the default ClientTimeout if no timeout was given. 58b7e3c